### PR TITLE
Add wayland-compatible on-screen keyboard installation instructions

### PR DESCRIPTION
**Pull request by nathan-contino**
* Closes https://github.com/raspberrypi/documentation/issues/3171
* GND -> `GND`, because it isn't an acronym.
* Minor heading and stylistic changes
* 70 + 50 = 120, QED

Considered adding usage instructions, but there isn't much to do. Looks like this on-screen keyboard shows up as a device and Just Works.

### CODE REVIEW

**Review by aallan — APPROVED** [review]

**Comment by aallan** [comment]
Has conflicts that need resolving.

File: documentation/asciidoc/accessories/display/display_intro.adoc [hunk 1]
```diff
@@ -1,17 +1,17 @@
 == Raspberry Pi Touch Display
 
-The https://www.raspberrypi.com/products/raspberry-pi-touch-display/[Raspberry Pi Touch Display] is an LCD display that connects to the Raspberry Pi using the DSI connector. While the panel is connected, you can use both it and the the normal HDMI display output at the same time.
+The https://www.raspberrypi.com/products/raspberry-pi-touch-display/[Raspberry Pi Touch Display] is an LCD display that connects to the Raspberry Pi using the DSI connector. You can use both the Touch Display and an HDMI display output at the same time.
 
 .The Raspberry Pi 7-inch Touch Display
 image::images/display.png[The Raspberry Pi 7-inch Touch Display, width="70%"]
 
-The Touch Display functions with all models of Raspberry Pi. However, the earliest Raspberry Pi models lack appropriate mounting holes, requiring additional mounting hardware to fit the stand-offs on the display PCB.
+The Touch Display is compatible with all models of Raspberry Pi. However, the earliest Raspberry Pi models lack appropriate mounting holes, requiring additional mounting hardware to fit the stand-offs on the display PCB.
```
Comment (lurch) [inline documentation/asciidoc/accessories/display/display_intro.adoc:8]: ...except the Raspberry Pi Zero and Zero 2 W, which lack the DSI connector?

Comment (nathan-contino) [reply documentation/asciidoc/accessories/display/display_intro.adoc:8]:
```suggestion
The Touch Display is compatible with all models of Raspberry Pi except the Raspberry Pi Zero and Zero 2 W, which lack a DSI connector. The earliest Raspberry Pi models lack appropriate mounting holes, requiring additional mounting hardware to fit the stand-offs on the display PCB.
```
Good call.

File: documentation/asciidoc/accessories/display/display_intro.adoc [hunk 2]
```diff
@@ -29,27 +29,9 @@ The display has the following key features:
...
-
-   Option "EmulateThirdButton" "1"
-   Option "EmulateThirdButtonTimeout" "750"
-   Option "EmulateThirdButtonMoveThreshold" "30"
-EndSection
-----
-
-These features are not available when running Raspberry Pi OS Bookworm.
-====
+WARNING: The Raspberry Pi 5 does not support automatic display configuration. To attach a display to the `CAM/DISP 1` connector, add `dtoverlay=vc4-kms-dsi-7inch` to `/boot/firmware/config.txt`. To attach a display to the `CAM/DISP 0` connector, add `dtoverlay=vc4-kms-dsi-7inch,dsi0`. Reboot with `sudo reboot` to load the new configuration.
```
Comment (lurch) [inline documentation/asciidoc/accessories/display/display_intro.adoc:32]: I believe this **is** now working automatically, with no need for config.txt edits.

Comment (nathan-contino) [reply documentation/asciidoc/accessories/display/display_intro.adoc:32]:
```suggestion
```
Excellent, thanks! Got caught up in a merge conflict and I didn't want to chance losing this if it was important. Turns out we removed it for a reason.

**Pull request merged**